src/data_type/age.verification.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const Validation = require('../yoti_common/validation');
+
+/**
+ * Wraps an 'Age Verify/Condition' attribute to provide behaviour specific
+ * to verifying someone's age.
+ *
+ * @class AgeVerification
+ */
+class AgeVerification {
+  constructor(attribute) {
+    Validation.notNull(attribute, 'attribute');
+    Validation.matchesPattern(attribute.getName(), /^[^:]+:(?!.*:)[0-9]+$/, 'attribute.name');
+    this.attribute = attribute;
+
+    const split = attribute.getName().split(':');
+    this.checkType = split[0];
+
+    this.age = parseInt(split[1], 10);
+    this.result = attribute.getValue() === 'true';
+  }
+
+  /**
+   * The type of age check performed, as specified on Yoti Hub. Currently this might
```

Comment:
I'm assuming you might have got this from another SDK, but I'm not a fan of using the word "currently", as it'll quickly get outdated! I'm more of a fan of taking the approach we do with sources i.e. `among the possible options are age_under and age_over`

Reply:
Updated in 22a7aec